src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -244,6 +244,12 @@ class ReferralService(
         existingDetails?.furtherInformation,
         existingDetails?.maximumEnforceableDays,
       )
+      handleChangLog(
```

Comment:
_typo_ (was present in the original too, but since we're moving it...)
```diff
- handleChangLog
+ handleChangeLog
```

But since we are at it, can we name it more according to what it does?

E.g. maybe `appendToChangelog` or `logChanges`?

Reply:
sure I like logChanges. I will rename that